pytorch_toolkit/tests/run_model_templates_tests.py

```diff
@@ -25,7 +25,10 @@
     ],
     'text_spotting_2': [
         'alphanumeric_text_spotting',
```

Comment:
please comment it, I'll try to figure out what is going wrong.

Why don't you use the latest commit of deep-object-reid?

Reply:
Ok, will disable this test. I was using an experimental commit from my branch of deep-object-reid, now I have to merge it to 
the ote branch and change the commit here.

Reply:
Tests are passed, deep-object-reid was updated.